src/descriptors/edit_structs.json

```diff
@@ -106,7 +106,7 @@
             {
                 "name": "file",
                 "comment": "mapping .csv file name in the coremaps directory",
-                "data_type": "String"
+                "data_type": "Any"
```

Comment:
**Generality**. Mapping now works given a DataFrame instead of a path to a .csv.